doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -92,17 +92,25 @@ Bluetooth Mesh
 Matter
 ------
 
+* From |NCS| 2.6.0 version the Arm PSA Crypto API is used as cryptography for devices that use Matter over Thread.
```

Comment:
```suggestion
* For devices that use Matter over Thread, the default cryptography backend is now Arm PSA Crypto API instead of Mbed TLS, which was used in earlier versions.
```